src/styles/reset.css

```diff
@@ -0,0 +1,8 @@
+* {
+  margin: 0;
```

Comment:
Everything except box sizing is forbidden inside `*` selector

Reply:
not resolved, add `box-sizing: border-box;` here, remove from other places and add `margin: 0` to body